precompile/contracts/deployerallowlist/simulated_test.go

```diff
@@ -70,170 +59,145 @@ func newBackendWithDeployerAllowList(t *testing.T) *sim.Backend {
-				setAsAdmin(t, backend, allowList, admin, allowListTestAddr)
-				verifyRole(t, allowList, allowListTestAddr, allowlist.AdminRole)
+				testutils.VerifyRole(t, allowList, allowlistbindingsAddr, allowlist.NoRole)
+				testutils.SetAsAdmin(t, backend, allowList, admin, allowlistbindingsAddr)
+				testutils.VerifyRole(t, allowList, allowlistbindingsAddr, allowlist.AdminRole)
 
-				tx, err := allowListTest.SetEnabled(admin, otherContractAddr)
+				tx, err := allowlistbindings.SetEnabled(admin, otherContractAddr)
 				require.NoError(t, err)
-				waitReceipt(t, backend, tx)
+				testutils.WaitReceipt(t, backend, tx)
 
-				isEnabled, err := allowListTest.IsEnabled(nil, otherContractAddr)
+				isEnabled, err := allowlistbindings.IsEnabled(nil, otherContractAddr)
 				require.NoError(t, err)
 				require.True(t, isEnabled)
-				verifyRole(t, allowList, otherContractAddr, allowlist.EnabledRole)
+				testutils.VerifyRole(t, allowList, otherContractAddr, allowlist.EnabledRole)
 			},
 		},
 		{
 			name: "should allow enabled address to deploy contracts",
-			test: func(t *testing.T, backend *sim.Backend, allowList *allowlisttest.IAllowList) {
-				allowListTestAddr, allowListTest := deployAllowListTestContract(t, backend, admin)
-				deployerContractAddr, deployerContract := deployAllowListTestContract(t, backend, admin)
+			test: func(t *testing.T, backend *sim.Backend, allowList *allowlistbindings.IAllowList) {
+				allowlistbindingsAddr, allowlistbindings := deployallowlistbindings(t, backend, admin)
+				deployerContractAddr, deployerContract := deployallowlistbindings(t, backend, admin)
 
-				setAsAdmin(t, backend, allowList, admin, allowListTestAddr)
+				testutils.SetAsAdmin(t, backend, allowList, admin, allowlistbindingsAddr)
 
-				tx, err := allowListTest.SetEnabled(admin, deployerContractAddr)
+				tx, err := allowlistbindings.SetEnabled(admin, deployerContractAddr)
 				require.NoError(t, err)
-				waitReceipt(t, backend, tx)
+				testutils.WaitReceipt(t, backend, tx)
 
-				isEnabled, err := allowListTest.IsEnabled(nil, deployerContractAddr)
+				isEnabled, err := allowlistbindings.IsEnabled(nil, deployerContractAddr)
 				require.NoError(t, err)
 				require.True(t, isEnabled)
 
 				tx, err = deployerContract.DeployContract(admin)
 				require.NoError(t, err)
-				receipt := waitReceipt(t, backend, tx)
-				require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status)
+				testutils.WaitReceiptSuccessful(t, backend, tx)
 			},
 		},
 		{
 			name: "should allow admin to revoke deployer",
-			test: func(t *testing.T, backend *sim.Backend, allowList *allowlisttest.IAllowList) {
-				allowListTestAddr, allowListTest := deployAllowListTestContract(t, backend, admin)
-				deployerContractAddr, _ := deployAllowListTestContract(t, backend, admin)
+			test: func(t *testing.T, backend *sim.Backend, allowList *allowlistbindings.IAllowList) {
+				allowlistbindingsAddr, allowlistbindings := deployallowlistbindings(t, backend, admin)
```

Comment:
can we revert the var name change? (i think a search and replace changed those in the PR 1878, should not match the upper/lower case here.)